### ISSUE TITLE
Feature Request: Port jc to nushell

### ISSUE THREAD

**Issue by Yethal** [nushell/nushell#3957]
### Related problem

jc (https://github.com/kellyjonbrazil/jc) is a tool that parses output of multiple standard unix commands into json for easier parsing using tools such as jq or nushell's own from json

### Describe the solution you'd like

Parser from jc project ported over to nushell so that whenever a supported command is launched using nu its output is automatically parsed into a table

### Describe alternatives you've considered

run the command as usual, pipe it into jc and then pipe the output to from json

**Comment by weirdan** [comment]
There's a `jc` wrapper module in `nu_scripts`: https://github.com/nushell/nu_scripts/tree/main/modules#jc

**Comment by kubouch** [comment]
Yeah, I think this type of usage is best covered by writing Nushell wrappers. We probably don't want this to be baked into Nushell's core.
